client/src/main/java/io/pravega/client/stream/impl/ReaderGroupState.java

```diff
@@ -114,9 +114,9 @@ int getNumberOfReaders() {
      */
```

Comment:
In the comment about the ranking, can we say that this is ranked according to descending order?

Reply:
done